.bcr/presubmit.yml

```diff
@@ -1,11 +1,16 @@
 matrix:
+  # Note that gcc 9 and earlier aren't supported due to using --std=c++2a
+  # instead of --std=c++20
   platform:
-  - centos7
-  - debian10
-  - ubuntu2004
+  - debian11
+  - ubuntu2024
```

Comment:
This should probably be 2404 instead of 2024? 2204 might also work since that has a relatively recent gcc.

Reply:
Thanks! That's a great catch!

Reply:
I meant `ubuntu2204`